### ISSUE TITLE
we should export SherpaVerbosity in the ui layers

### ISSUE THREAD

**Issue by DougBurke** [sherpa/sherpa#1588]

**Comment by hamogu** [comment]
Of course, we *could* do that, and it won't be hard to do. However, I'm not sure it's needed. the UI layer already has a A LOT of symbols in it and this might be an opportunity to subtly tell our users "While we make it look like XSPEC, this is, in fact, Python. And see here what you can get by using real Python commands."

Maybe we find an opportunity to work that into any threads or examples.
